game_config.yml

```diff
@@ -0,0 +1 @@
+dlgr/griduniverse/game_config.yml
```

Comment:
@silviot I think I fixed the issue that required this yesterday on the `one-hour-one-life` main branch.